src/main/kotlin/lotto/domain/LottoStore.kt

```diff
@@ -1,11 +1,16 @@
 package lotto.domain
 
-class LottoStore(private val userMoney: UserMoney, private val lottoMaker: LottoMaker = KoreanLottoNumberMaker()) {
-    val lottoCount = userMoney.getLottoCount()
-    val boughtLottos = List(lottoCount) { lottoMaker.makeLottoNumbers() }
+class LottoStore(
+    private val money: Money,
+    private val manualCount: Int,
```

Comment:
프로퍼티로 들고있지 않아도 될 것 같아요!

```suggestion
    manualCount: Int,
```